src/algorithm/MEMCPY-OMPTarget.cpp

```diff
@@ -90,4 +92,4 @@ void MEMCPY::runOpenMPTargetVariant(VariantID vid, size_t RAJAPERF_UNUSED_ARG(tu
 } // end namespace algorithm
 } // end namespace rajaperf
 
-#endif  // RAJA_ENABLE_TARGET_OPENMP
+#endif  // RAJA_ENABLE_TARGET_OPENMP
```

Comment:
What about these no newline at end of file formatting changes?

Reply:
@MrBurmark do you want to push changes to fix those. Not sure if @ThorBl has permissions to do that.